pkg/plugins/mercury/transmitter.go

```diff
@@ -0,0 +1,11 @@
+package mercury
+
+import (
+	"context"
+
+	ocrtypes "github.com/smartcontractkit/libocr/offchainreporting2/types"
+)
+
+type Transmitter interface {
+	Transmit(ctx context.Context, reportCtx ocrtypes.ReportContext, report ocrtypes.Report, signatures []ocrtypes.AttributedOnchainSignature) error
```

Comment:
Why does it need a custom transmitter? Isn't this the same signature as the `ContractTransmitter`?

Reply:
Correct, fixed